packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -375,7 +375,7 @@ module.exports = {
       // used to populate the caches, to ensure the responses are fresh.
       // If a URL is already hashed by Webpack, then there is no concern
       // about it being stale, and the cache-busting can be skipped.
-      dontCacheBustUrlsMatching: /\.\w{64}\./,
+      dontCacheBustUrlsMatching: /./,
```

Comment:
Nice! Do we need this property at all?

Reply:
yes, without it defaults to cache busting everything.

Reply:
I'm not 100% sure on this one. Why is this a problem for us and not for any other CRA app?

Reply:
CRA defaults assume that people don't have the correct (or any) set `cache-control` (and related) headers properly on the web server where the static content comes from.
In those cases the cache busting is the  "safe option" for making sure that fresh content always gets the browsers at the cost nothing being cached.

We have the correct headers set, and therefore can turn off the cache busting so that browser (and intermediate server) caches are used.

Reply:
actually to be more correct it's `SWPrecacheWebpackPlugin` that assumes people's cache headers are all wrong.
CRA itself does shorter version of  hashing like we do for the static assets, and sets this value to `/\.\w{8}\./` which basically turns off cache busting for the static assets (because if those assets change, so does their name and therefore no risk of the them being in a cache.)